PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/utility/AccessPrivate.java

```diff
@@ -1,4 +1,4 @@
-package nl.tudelft.pl2016gr2.test.utility;
```

Comment:
@cbilstra we should definitely not move it to `src/test` because that is where the test classes reside.

We could consider a different folder name, because I agree it can be somewhat confusing.
I would like the folder structure to have the distinction of:
1) these classes are not ours, they're 3rd party
2) these classes are utilities for **testing** only.

My suggestions:
`nl.tudelft.pl2016gr2.thirdparty.testing.utility`
`nl.tudelft.pl2016gr2.thirdparty.testingutility`

`testing` is different from `test` (it is for 'testing', rather than it is a 'test')